frontend/src/app/user/verify-email/verify-email.component.html

```diff
@@ -0,0 +1,27 @@
+<h1>Confirm e-mail address</h1>
+
+<div *ngIf="userDetails$ | async as userDetails" class="mb-3">
+    <p>
+        Please confirm that
+        <a href="mailto:{{userDetails.email}}">{{userDetails.email}}</a>
+        is an e-mail address for user {{userDetails.username}}.
+    </p>
+
+    <button class="btn btn-primary" (click)="confirm()">Confirm</button>
+</div>
+
+<div *ngIf="error$ | async as error" class="alert alert-danger" role="alert">
+    <h2 class="h6 alert-heading">Error</h2>
+    {{error.error}}
+</div>
+
+<div *ngIf="success$ | async" class="alert alert-success" role="alert">
+    <h2 class="h6 alert-heading">E-mail confirmed</h2>
+    Your email address has been verified!
+    <a *ngIf="directToLogin; else homeLink" [routerLink]="['/login']" class="alert-link">
+        Sign in
+    </a>
+    <ng-template #homeLink>
```

Comment:
I now realise the new Angular 16+ control-flow syntax would really shine here by getting rid of those silly `ng-templates`:

```html
@if (directToLogin) {
    <a [routerLink]="...">Sign in</a>
} @else {
    <a [routerLink]="...">Return to home page</a>
}
```